WordPress/src/main/java/org/wordpress/android/ui/layoutpicker/LayoutsRowViewHolder.kt

```diff
@@ -56,6 +56,7 @@ class LayoutsItemViewHolder(
                     RecyclerView.HORIZONTAL,
                     false
             ).apply { initialPrefetchItemCount = prefetchItemCount }
+            // TODO: use a shared instance of the pool (or alternatively, remove this?)
```

Comment:
I left this comment as a reminder, but this isn't really tied to this PR. Currently, we have 20 designs over 7 categories from the endpoint. Though these numbers will soon increase, I wonder if we get much benefit from sharing a pool? We seem to have been ok in the current state (which I think is the same if we remove this line), i.e. each nested row has it's own pool of recyclable views. I'm thinking unless we have a lot more, maybe we can simply remove this line? Wdyt?

Reply:
That sounds reasonable to me. I've tested with removing the line and it still works as expected.
Btw the `RecycledViewPool` is a `static` class. I'm not sure if creating an instance makes a big difference 🤔

Reply:
> Btw the `RecycledViewPool` is a `static` class. I'm not sure if creating an instance makes a big difference

Though the _class_ is static, unlike static members, the nested class can have multiple instances (i.e. `RecycledViewPool` does not behave like a singleton). In this case, being static "frees" it from the enclosing class (`RecyclerView`).

Reply:
I removed this line with: https://github.com/wordpress-mobile/WordPress-Android/pull/16584/commits/b2f99c7240e2de40b03b0a82808893cefc37754c.